environments/php7/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM php:7.1-cli
+
+RUN apt-get update && apt-cache search libjpeg && apt-get install -y libjpeg-dev && apt-get install -y \
```

Comment:
apt-cache is for searching package. it's not necessary. I will remove this command.